shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -166,6 +166,20 @@ - (FlutterTextRange*)getLineRangeFromTokenizer:(id<UITextInputTokenizer>)tokeniz
 
 #pragma mark - Tests
 
+- (void)testTextInputPluginWillNotHandleMethodWhenViewControllerIsNil {
+  FlutterEngine* flutterEngine = [[FlutterEngine alloc] init];
+  FlutterTextInputPlugin* inputPlugin =
+      [[FlutterTextInputPlugin alloc] initWithDelegate:(id<FlutterTextInputDelegate>)flutterEngine];
+  XCTAssertNil(inputPlugin.viewController);
+  FlutterMethodCall* methodCall = [FlutterMethodCall methodCallWithMethodName:@"TextInput.show"
+                                                                    arguments:nil];
+  [inputPlugin handleMethodCall:methodCall
+                         result:^(id _Nullable result){
```

Comment:
you may wanna validate that the result is actually called with nil argument.

Reply:
Done O(∩_∩)O